src/Content/Widget.php

```diff
@@ -526,8 +522,30 @@ public static function postedByYear(string $url, int $uid, bool $wall)
 		return $o;
 	}
 
+	/**
+	 * Display the account types sidebar
+	 * The account type value is added as a parameter to the url
+	 * 
+	 * @param string $base        Basepath
+	 * @param int    $accounttype Acount type
+	 * @return string
+	 */
+	public static function accounttypes(string $base, $accounttype)
+	{
+		$accounts = [
+			['ref' => 'person', 'name' => DI::l10n()->t('Persons')],
+			['ref' => 'organisation', 'name' => DI::l10n()->t('Organisations')],
+			['ref' => 'news', 'name' => DI::l10n()->t('News')],
+			['ref' => 'community', 'name' => DI::l10n()->t('Forums')],
+		];
+
+		return self::filter('accounttype', DI::l10n()->t('Accounts'), '',
```

Comment:
```suggestion
		return self::filter('accounttype', DI::l10n()->t('Account Types'), '',
```

Reply:
Optically the two widget should look similar. They only behave differently concerning the way the parameters are set. So do you suggest replacing "Account" in both widgets with "Account Types"?

Reply:
I would prefer that.